wallet/wallet-cli-lib/src/config.rs

```diff
@@ -135,6 +135,22 @@ pub struct CliArgs {
     /// Enable running the wallet service without RPC authentication (assuming wallet RPC server is enabled)
     #[arg(long, conflicts_with_all(["wallet_rpc_password", "wallet_rpc_username", "wallet_rpc_cookie_file"]))]
     pub wallet_rpc_no_authentication: bool,
+
+    /// Optional RPC address
+    #[arg(long, conflicts_with_all(["wallet_rpc_password", "wallet_rpc_username", "wallet_rpc_cookie_file", "wallet_rpc_no_authentication", "wallet_rpc_bind_address", "node_rpc_address", "node_rpc_cookie_file", "node_rpc_username", "node_rpc_password"]))]
+    pub connect_to_rpc_wallet_address: Option<String>,
```

Comment:
You can define an "option group" in clap to specify a collection of options. I believe you can then use te group in the `conflicts_with*` calls and similar. See `wallet-rpc-lib/src/cmdline.rs`.